sites/docs/src/content/docs/contributing/project_proposals.md

```diff
@@ -0,0 +1,163 @@
+</dl>
+
+:::note
+**You are not alone as a champion!** If this is your first time writing an RFC or design document, our maintainer team will be available to work with you and guide you through this process.
+:::
+
+## Scope
+
+Not every project in nf-core requires an RFC. The process should only be used for efforts that will affect a significant proportion of community members. For example:
+
+- Changes to established development / code standardisation
+- Creation of new nf-core fundamental product / initiative
+- Changes to base dependencies and requirements that span many pipelines
+
+:::tip{collapse title="Examples"}
+
+To help give a better idea of the scale that a request should reach to require an RFC, we provide a few real-life examples here (from before we had an RFC procedure):
+
+- nf-core/reports
+  - We would like to add a new ‘product’ offered by and for the community in the form of standardised analysis notebooks for routine post-pipeline analyses that require more user experimentation. This would require new subcommands of nf-core/tools, a new template, new repositories, and a new section on the top bar of the nf-core website.
+- nf-core/tools test datasets
+  - I would like to propose a new subcommand to the nf-core tools package, called `nf-core testdatasets`. The purpose of this subcommand is to make it easier for developers to search through the very large and convoluted nf-core/test-datasets repository for suitable test-data files to promote reuse of existing data files.
+- Standardise `ext.args` naming
+  - There is a deficiency within the nf-core/modules specifications regarding standardised use of the `ext.args` variable. The use of the naming scheme `args`, `args2`, `args3` are not intuitive for developers to know which `arg` goes to which command in the module. We propose to update the specifications to use the command/subcommand as a suffix instead e.g., `args_samtoolssort`. This would require a change to every module in the repository and improved documentation support.
+
+:::
+
+As a rule of thumb, if your proposal will require changes across two or more different repositories, it is a good candidate for an RFC.
+Projects that are smaller in scope should typically be raised as a new issue on the relevant GitHub repository instead.
+
+# Stage 1: Suggestion (optional)
+
+### Goal
+
+Provide a space for informal, low-barrier discussions around ideas and potential improvements to nf-core. This stage is useful for gathering early feedback and assessing community interest from the community, maintainers, and other stakeholders.
```

Comment:
```suggestion
Provide a space for informal, low-barrier discussions around ideas and potential improvements to nf-core.
This stage is useful for gathering early feedback and assessing community interest from the community, maintainers, and other stakeholders.
```

One line per sentence as you tell me to do @ewels ;)

Reply:
Hah, copied as Markdown from Google Docs in this case - can you tell? 😅